lfs/scanner.go

```diff
@@ -16,7 +16,7 @@ import (
 const (
 	// blobSizeCutoff is used to determine which files to scan for Git LFS
 	// pointers.  Any file with a size below this cutoff will be scanned.
-	blobSizeCutoff = 140
```

Comment:
I opened an issue to turn this into a dynamic setting based on the repo config: https://github.com/github/git-lfs/issues/524